src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -70,6 +70,13 @@ public String getResponseTextBoxValue(int qnNumber, int responseNumber) {
         return element.getAttribute("value");
     }
 
+    public String getResponseTextBoxValue(int qnNumber, int responseNumber, int responseSubNumber) {
+        WebElement element = browser.driver.findElement(
+                By.id(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT
+                    + "-" + qnNumber + "-" + responseNumber + "-" + responseSubNumber));
```

Comment:
@chowyb other functions are implemented in this manner only. I think we should open a seperate issue for that to improve code readability.

Or should I just correct this?